### PR TITLE
Update chplspell dictionaries

### PR DESCRIPTION
**Pull request by cassella**
Update chplspell dictionaries

Without the change to `gpuTransforms.cpp`, the dictionary would need to include `uization` because it treats "GPUization" as GP + Uization. Thankfully the capitalization of "gpuization" is a better fit with the other occurrences of this word. While the dictionary already contains "ufunction" and "lcode" and some others in order to accept "CUfunction" and "CURLcode" et al. for the same reason, these are more benign than "uization" and also we cannot change them.

### CODE REVIEW

**Comment by cassella** [comment]
@vasslitvinov Could you review?

File: util/devel/chplspell-dictionary [hunk 1]
```diff
@@ -4730,6 +4735,7 @@ fossa
 indri
 kitware
 kudu
+mantic
```
Comment (vasslitvinov) [inline util/devel/chplspell-dictionary:4738]: Where is `mantic` from?

File: util/devel/chplspell-dictionary [hunk 2]
```diff
@@ -5641,7 +5655,9 @@ acct
 
 FILEID: e72aeca4-45ab-11ec-a187-336475c959d4
 doit
+gpuized
 griddim
+uization
```
Comment (vasslitvinov) [inline util/devel/chplspell-dictionary:5660]: "uization" does not sound right. ???

File: util/devel/chplspell-dictionary (hunk 1 above)
Comment (cassella) [reply util/devel/chplspell-dictionary:4738]:
```
doc/rst/usingchapel/prereqs.rst:
  * Ubuntu 22.04 "Jammy Jellyfish", 23.10 "Mantic Minotaur"::
```

File: util/devel/chplspell-dictionary (hunk 2 above)
Comment (cassella) [reply util/devel/chplspell-dictionary:5660]: That's from

```
./compiler/optimizations/gpuTransforms.cpp:
    // GPUization failed (it's an implementation detail). Instead, we want
```

I tend to err on the side of accepting what's written unless it's objectively wrong.  Another option would be to change that comment to `Gpuization` or `gpuization`, which are both used elsewhere in the Chapel source.

Comment (cassella) [reply util/devel/chplspell-dictionary]: I changed the comment to "gpuization" per offline discussion.

**Pull request merged**